### PR TITLE
likes incremention added

### LINKED ISSUE #41: Bug report
**Describe the bug**
The Like button is not working 

**To Reproduce**
- Steps to reproduce the behavior:
1. We can use react hook useState to increment likes (even for the dislike) 

**Expected behavior**
- It will display total no of likes for each product

**Screenshots**
issue2

**Additional details**
I am contributor at gssoc'22
Please assign this task to me.

### PR DESCRIPTION
**Pull request by Siddhesh777**
Implemented the useState hook to increment the likes .
Kindly review.

Closes #41

### CODE REVIEW

**Comment by Siddhesh777** [comment]
hey @adityabisoi 
Can you please review this PR?

**Comment by pranav2012** [comment]
> hey @adityabisoi Can you please review this PR?

Hey try to save the state in local storage as well so it will maintain it even after refresh

**Comment by Siddhesh777** [comment]
hey @pranav2012 
I did it !
Can you please review it !

File: client/src/Components/Product/ProductPreview.jsx [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
-const ProductPreview = ({id, heading ,description ,image ,likes }) => {
-    let linkStyle = {border:'1px solid orange', padding: '0.3em', borderRadius:'5px'};
+const ProductPreview = ({ id, heading, description, image, likes }) => {
+  let linkStyle = {
+    border: "1px solid orange",
+    padding: "0.3em",
+    borderRadius: "5px",
+  };
+  const [currentLikes, setCurrentLikes] = React.useState(likes);
+  const str = localStorage.getItem("dummyProducts");
```
Comment (pranav2012) [inline client/src/Components/Product/ProductPreview.jsx]: instead of const variable make it a state will be more efficient

Comment (pranav2012) [reply client/src/Components/Product/ProductPreview.jsx]: @Siddhesh777 make this final changes everything else LGTM!

**Comment by Siddhesh777** [comment]
hey @pranav2012 
Can you please review It ?

**Review by pranav2012 — APPROVED** [review]

**Comment by Siddhesh777** [comment]
hey @pranav2012 
Can you please merge it?

**Comment by pranav2012** [comment]
> hey @pranav2012 Can you please merge it?

@adityabisoi will merge it after final review

**Pull request merged**